packages/gatsby/src/utils/worker/__tests__/share-state.ts

```diff
@@ -208,18 +208,23 @@ describe(`worker (share-state)`, () => {
 
     saveStateForWorkers([`components`, `staticQueryComponents`])
 
-    await worker.setQueries()
+    await worker.single.setQueries()
 
-    const components = await worker.getComponent(dummyPagePayload.component)
-    const staticQueryComponents = await worker.getStaticQueryComponent(
+    const components = await worker.single.getComponent(
+      dummyPagePayload.component
+    )
+    const staticQueryComponents = await worker.single.getStaticQueryComponent(
       staticQueryID
     )
 
     expect(components).toMatchInlineSnapshot(`
       Object {
+        "componentChunkName": undefined,
         "componentPath": "/foo",
         "isInBootstrap": true,
-        "pages": Object {},
+        "pages": Set {
+          "/foo/",
+        },
```

Comment:
`Set` and `Map` work with `worker_threads` 's `postMessage` (but it would work with `jest-worker` in `worker_theads` mode as well